src/impls.rs

```diff
@@ -126,6 +127,17 @@ where
     }
 }
 
+impl<T> TypeInfo for VecDeque<T>
+where
+    T: TypeInfo + 'static,
+{
+    type Identity = Vec<T>;
```

Comment:
Why not `type Identity = [T];` here?

Reply:
You are right, it should be this in order that `[T]`, `Vec<T>` and `VecDeque<T>` all point to the same type

Reply:
https://github.com/paritytech/scale-info/pull/100